tests/Action/AppendFormFieldElementActionTest.php

```diff
@@ -60,14 +60,12 @@ protected function setUp(): void
     {
         $this->twig = $this->createStub(Environment::class);
         $this->admin = $this->createMock(AbstractAdmin::class);
-        $this->admin->expects($this->once())->method('setRequest');
-        $container = new Container();
-        $container->set('sonata.post.admin', $this->admin);
-        $this->pool = new Pool($container, ['sonata.post.admin']);
+        $this->adminFetcher = $this->createMock(AdminFetcherInterface::class);
```

Comment:
```suggestion
        $this->adminFetcher = $this->createStub(AdminFetcherInterface::class);
```
I think all `$this->createMock(AdminFetcherInterface::class)` could be replaced by `$this->createStub(AdminFetcherInterface::class)`

Reply:
Indeed, done.